Releasenotes/Releasenotes.md

```diff
@@ -1,3 +1,9 @@
+**Release notes 3.0.0:**  (12 juli 2023)
```

Comment:
Versie 3.0.0 is t.o.v. 2.9.0 een nieuwe major versie, maar dat kan niet de bedoeling zijn. Als dat zo zou zijn, zouden we versie 2.9.0 en 3.0.0 naast elkaar moeten aanbieden. M.i. is dit een bug fix: de implementatie voldeed niet aan de specificatie, dus zou het een versie 2.9.1 moeten zijn (als het alleen deze wijziging betreft).